helm-repositories/banzaicloud-kafka-repo/banzaicloud-kafka-repo.yaml

```
@@ -0,0 +1,8 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmRepository
+metadata:
+  name: kubernetes-charts.banzaicloud.com-kafka-charts
```

```suggestion
  name: kubernetes-charts.banzaicloud.com
```

Yeah I think we should name it generically by the repo name!

---

fixed in 71e46fad23bc316b9bc0a8c12bb9e61ded9849a2!